src/AryUtil.chpl

```diff
@@ -5,6 +5,14 @@ module AryUtil
     use Reflection;
     use Logging;
     use ServerConfig;
+    use MultiTypeSymbolTable;
+    use MultiTypeSymEntry;
+    use ServerErrors;
+    use BitOps;
+
+    param bitsPerDigit = RSLSD_bitsPerDigit;
```

Comment:
@reuster986 and I talked about reworking the `RadixSortLSD` code layout between modules. I would like to see it refactored into just two modules maybe call them `RadixSortLSD_core` and `RadixSortLSD_support`. This would merge the stuff from this file and `ServerConfig` into `RadixSort_support`, so all the helper functions and parameters. And we would switch to using ``private config param` for commandline parameters.

Reply:
i'll add this to issue on refactor of sort code